c7n/loader.py

```diff
@@ -193,7 +192,13 @@ def _validate(data):
 
         def _load(directory, raw_policies, errors):
             raw_policies = []
-            for root, dirs, files in os.walk(directory, topdown=False):
+            for root, dirs, files in os.walk(directory):
+                if is_hidden(root):
```

Comment:
not sure that you really need the root check here since the dirs filtering below will prevent recursing into a hidden dir, unless directory itself is the hidden dir which was the users express intent at that point.